src/components/Clusters/views/ClusterOverview/ClusterValidation/ClusterValidation.tsx

```diff
@@ -0,0 +1,292 @@
+    [selectedConfiguration, defaultConfiguration],
+  );
+  const [isConfigurationOpen, setConfigurationOpen] = useState(false);
+  const configure = () => {
+    setConfigurationOpen(true);
+  };
+
+  const [currentScan, setCurrentScan] = useState<Scan | null>();
+
+  const [scanResult, _setScanResult] = useState<ScanResult | null>();
+  const setScanResult = (result: ScanResult | null) => {
+    if (result) _setScanResult({ ...result });
+    else _setScanResult(result);
+  };
+
+  const scanSettings = {
+    concurrentRequests: configuration?.scanParameters?.parallelRequests ?? 5,
+    concurrentWorkers: 1,
+    backpressureBuffer: 3,
+  };
+  resourceLoader.queue.concurrency = Math.max(
+    scanSettings.concurrentRequests,
+    1,
+  );
+
+  const [scanProgress, setScanProgress] = useState<ScanProgress | null>();
+
+  const scan = async () => {
+    if (!validationSchemas || !listableResources) return;
+    const enabledSchemas = getValidationEnabledSchemas(
+      validationSchemas,
+      new Set(configuration.policies),
+    );
+    const _currentScan = new Scan(resourceLoader, enabledSchemas);
+    setCurrentScan(_currentScan);
+    await ResourceValidation.setRuleset(enabledSchemas);
+
+    const resourcesToScan = listableResources;
+
+    await _currentScan.scan({
+      namespaces: configuration?.namespaces,
+      resources: resourcesToScan,
+      setScanProgress,
+      setScanResult,
+      post,
+      concurrency:
+        Math.max(
+          scanSettings.concurrentRequests,
+          scanSettings.concurrentWorkers,
+        ) + scanSettings.backpressureBuffer,
+    });
+  };
+
+  const clear = () => {
+    currentScan?.abort();
+    setCurrentScan(null);
+    setScanResult(null);
+    setScanProgress(null);
+  };
+
+  return (
+    <LayoutPanel className="fd-margin--md">
+      <LayoutPanel.Header className="fd-has-padding-left-small fd-has-padding-right-small">
+        <LayoutPanel.Head title={'Cluster Validation'} />
+        <LayoutPanel.Actions>
+          <Button
+            glyph="play"
+            onClick={scan}
+            disabled={!!scanProgress || !scanReady}
+          >
+            {t('cluster-validation.scan.buttons.scan')}
+          </Button>
+          <Button
+            glyph="settings"
+            onClick={configure}
+            disabled={!!scanProgress || !scanReady}
+          >
+            {t('cluster-validation.scan.buttons.configure')}
+          </Button>
+          <Button glyph="reset" onClick={clear} disabled={!scanProgress}>
+            {t('cluster-validation.scan.buttons.clear')}
+          </Button>
+        </LayoutPanel.Actions>
+      </LayoutPanel.Header>
+
+      {!scanReady && (
+        <LayoutPanel.Filters className="fd-has-padding-none">
+          <Loader type="Indeterminate" />
+        </LayoutPanel.Filters>
+      )}
+
+      <LayoutPanel.Body>
+        <Section titleText={t('cluster-validation.scan.scope')}>
+          <FlexBox>
```

Comment:
Consider adding padding to the flexbox
```suggestion
                    <FlexBox className="fd-has-padding-small">
```